src/util/system.cpp

```diff
@@ -30,7 +30,7 @@ void system_quick_exit(int status) {
 #ifdef WINDOWS_ENABLED
 	TerminateProcess(GetCurrentProcess(), status);
 #else
-	std::quick_exit(status);
```

Comment:
Can an `ifdef` be added? Also pass the status code.

Reply:
Ok, I had to look for a bit for the right flag but `_GLIBCXX_HAVE_QUICK_EXIT` does it i think.